mq/util.go

```diff
@@ -79,11 +78,11 @@ func publish(host *models.Host, dest string, msg []byte) error {
 	if encryptErr != nil {
 		return encryptErr
 	}
-	if mqclient == nil || !mqclient.IsConnectionOpen() {
+	if mqclient == nil || !mqclient.IsConnected() {
 		return errors.New("cannot publish ... mqclient not connected")
 	}
 
-	if token := mqclient.Publish(dest, 0, true, encrypted); !token.WaitTimeout(MQ_TIMEOUT*time.Second) || token.Error() != nil {
+	if token := mqclient.Publish(dest, 0, true, encrypted); token.Wait() && token.Error() != nil {
```

Comment:
`func (mqtt.Token) Wait() bool
Wait will wait indefinitely for the Token to complete, ie the Publish to be sent and confirmed receipt from the broker.`

we shouldn't wait forever, there should be a timeout, what's the thought process behind this?